test/SmokeTests/OOProcSmokeTests/durableJava/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id "com.microsoft.azure.azurefunctions" version "1.9.0"
+}
+apply plugin: 'java'
+apply plugin: "com.microsoft.azure.azurefunctions"
+
+group 'com.functions'
+version '0.1.0-SNAPSHOT'
+
+repositories {
+    mavenLocal()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.microsoft:durabletask-client:1.0.0-beta.1'
```

Comment:
Can this line be removed since it's transitively included with the `durabletask-azure-functions` dependency?